XpringKit.podspec

```diff
@@ -6,15 +6,15 @@ Pod::Spec.new do |spec|
   spec.description  = "
     XpringKit provides a Swift based SDK for interacting with Xpring Platform (XRP/ILP)
 
-    XpringKit is part of the Xpring SDK. To learn more, visit: http://github.com/xpring-eng/Xpring-SDK.
+    XpringKit is part of the Xpring SDK. To learn more, visit: http://github.com/dangell7/Xpring-SDK.
   "
 
   spec.homepage     = "http://xpring.io"
   spec.license      = "MIT"
   spec.author       = { "Xpring Engineering" => "xpring@ripple.com" }
-  spec.source       = { :git => "https://github.com/xpring-eng/XpringKit.git", :tag => "1.2.1" }
+  spec.source       = { :git => "https://github.com/dangell7/XpringKit.git", :tag => "1.2.1" }
```

Comment:
ditto

Reply:
Done. I'm still learning and this was me trying to get a development pod going. But I think I can just do assign a path in the pod file correct?

Reply:
> I'm still learning and this was me trying to get a development pod going.

Don't worry, thanks for the valuable feedback and the contribution. 

> But I think I can just do assign a path in the pod file correct?

I'm not sure I follow. You want to fork your own Pod from this one? ?